libraries/botbuilder-ai/src/luisRecognizer.ts

```diff
@@ -205,6 +210,33 @@ export class LuisRecognizer {
         });
     }
 
+    private prepareErrorMessage(error: Error): Error {
+        switch ((error as any).response.statusCode) {
+            case 401:
```

Comment:
The text in these responses doesn't match the text found on the LUIS docs site:
https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-reference-response-codes. 

Despite that I like your text better, I believe you should cut/paste the text from the docs and use that.

Reply:
Note: If you said "I cut a PR on the docs site to update their text to the next text" that would be the best choice.

Reply:
Due to the lack of knowledge around the docs' release cadence (iirc they like to gather a lot and release at once) and how there might be a diff for a while, I changed the 401 error message to more closely follow what was found in the docs' API code explanations. The new error message is this:

> Response 401: The key used is invalid, malformed, empty, or doesn't match the region.

I'm going to add @diberry to this conversation to see if we can update the docs in the future and then add those new explanations to the SDK.